gnomad_qc/v2/sample_qc/get_topmed_dups.py

```diff
@@ -12,7 +12,7 @@
 
 def create_shared_sites_table(data_type: str, overwrite: bool):
     freq_ht = hl.read_table(annotations_ht_path(data_type, 'frequencies'))
-    topmed_ht = hl.read_matrix_table('gs://gnomad-public/resources/hail-0.2/topmed.b37.mt').rows()
+    topmed_ht = hl.read_matrix_table('gs://gcp-public-data--gnomad/resources/hail-0.2/topmed.b37.mt').rows()
```

Comment:
I didn't find this one

```
$ gsutil ls gs://gcp-public-data--gnomad/resources/hail-0.2/topmed.b37.mt
CommandException: One or more URLs matched no objects.
```

Reply:
Yeah -- although now that I'm thinking about it, it may be better to leave it with the `gs://gnomad-public` root since that filepath is obviously deprecated. I'll change it back